### ISSUE TITLE
Beta UI: dropdown visibility and status in screen reader as unavailable

### ISSUE THREAD

**Issue by klml** [zammad/zammad#6163]
### Used Zammad Version

7.0.1-df77241a.docker

### Environment

Openshift

### Actual behaviour

The dropdown menus for visibility and status are read by the screen reader as “grayed out and unavailable,” but the options are still selectable.

<img width="302" height="136" alt="Image" src="https://github.com/user-attachments/assets/9f506247-3ffb-4531-98cd-b5630dcc203f" />

<img width="295" height="261" alt="Image" src="https://github.com/user-attachments/assets/381b7a65-fc96-49a1-a45c-5970831bb6ea" />

### Expected behaviour

Screen readers should be able to use the visibility status in some way, either via the inline dropdown or through an alternative method

### Steps to reproduce the behaviour

Use screenreader

### Support Ticket

7.0.1-df77241a.docker

### I'm sure this is a bug and no feature request or a general question.

no

**Comment by dominikklein** [comment]
@klml We currently do not understand what is not working. I tested it with NVDA, and for me all was working as expected, but I think in the report is something missing which points to the real problem.

Can you give some more information?

**Comment by dominikklein** [comment]
@klml Maybe you did not see my last message, but we can not really work on that without the additional information.

**Comment by klml** [comment]
@dominikklein sorry, I'm checking internally right now

**Comment by klml** [comment]
not reproducible (poosible drive by fixed in latest version)
